ChangeLog

```diff
@@ -1,6 +1,10 @@
 CHANGES
 =======
 
+4.1.3
```

Comment:
How does the versioning work in this project? I couldn't find any version string outside this text file. I also don't see any automated PyPi deploys. Do you do PyPi uploads manually, @dbaxa ?

Reply:
This project uses `pbr` which does the version bumping (somewhat) for us. I manually tweak the resulting changelog that pbr generates though.

Reply:
Cool, I'll leave that to you then.